content/blog/posts/student-story-doublej472.rst

```diff
@@ -0,0 +1,42 @@
+On Leaving the Open Source Lab, Jonathan Frederick
+--------------------------------------------------
+:author: Jonathan Frederick
+:date: 2019-09-25
+:img: doublej472-story.jpg
+:slug: student-story-doublej472
+:tag: featured-stories
+:tags: student-stories
+
+My first Linux operating system was Ubuntu 10.04, when I was still young, and it was my first experience with open
+source software, which I didn't fully appreciate at the time. I just wanted an operating system to work on my Dell
+hand-me-down computer and I didn't want to pay for windows, and Linux was the first thing I found. Over the next few
+years I started to really appreciate open source software and realized it's importance on the internet and for myself,
+since it was one of the only things that allows me to have full control over my devices.
+
+Around this time I learned about the Open Source Lab, since it provided hosting for many open source projects that I
+used. I preferred to use the OSL mirrors for downloading files from open source projects because they were the closest
+and fastest. I was very happy with the service they provided to those projects, allowing people like me to get into open
+source software.
+
+A little while later, I got accepted as a student at OSU. I started to look for jobs on the OSU website, and found that
+the OSL was hiring! I jumped on that oppourtunity after remembering how it provided a fast and close mirror for various
+open source software projects, and it lead me to working at the OSL as a student systems engineer.
+
+While working at the OSL I gained more experience than the classes could teach me. I was working on real projects and
+with real people. I felt like I was making a real impact and improving the open source projects I had used and even some
+projects I never heard of before, but soon came to know rather well.
+
+I had never experienced running a truly production Linux system before, and learning about how different things were
+from how I ran Linux taught me a mountain of information that I am grateful to have learned. I also was able to give a
```

Comment:
> I had never experienced running a truly production Linux system before, and learning about how different things were from how I ran Linux taught me a mountain of information that I am grateful to have learned.

You may want to break this long sentence into two more readable sentencesbelow:
`I had never experienced running a truly production Linux system before. Learning about how different things were from how I ran Linux taught me a mountain of information that I am grateful to have learned.`